formats/ome-converter-plugin/pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "polus-plugins-formats-ome-converter"
+version = "0.1.0"
+description = ""
+authors = ["Nick Schaub <nick.schaub@nih.gov>",
+           "hamshkhawar <hamdahshafqat.abbasi@nih.gov>"
+
+]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[[tool.poetry.source]]
+name = "test"
+url = "https://test.pypi.org/simple/"
+secondary = true
+
+[tool.poetry.dependencies]
+python = "^3.9"
+bfio = {version = "2.1.9", extras = ["all"]}
+filepattern = "^2.2.7"
+typer = "^0.7.0"
+requests = "^2.28.2"
+tqdm = "^4.64.1"
+scikit-image = "^0.19.3"
```

Comment:
These should be development dependencies. They are not needed to run the plugin.